src/components/data-table-v2/_data-table-v2-sort.scss

```diff
@@ -66,3 +66,118 @@
     min-width: 14px;
   }
 }
+
+@mixin data-table-v2-sort--x {
+  // -------------------------------------
+  // Sortable table
+  // -------------------------------------
+
+  .#{$prefix}--data-table-v2--sort th:hover {
+    background: $data-table-column-hover;
+    border-top: 1px solid $data-table-column-hover;
+    border-bottom: 1px solid $data-table-column-hover;
+  }
+
+  .#{$prefix}--data-table-v2--sort th.#{$prefix}--table-column-icon:hover,
+  .#{$prefix}--data-table-v2--sort th.#{$prefix}--table-column-menu:hover,
+  .#{$prefix}--data-table-v2--sort th.#{$prefix}--table-expand-v2:hover {
+    background: $ui-03;
+    border-top: 1px solid $ui-03;
+    border-bottom: 1px solid $ui-03;
+  }
+
+  .#{$prefix}--data-table-v2--sort th {
+    padding: 0;
+  }
+
+  .#{$prefix}--data-table-v2--sort th.#{$prefix}--table-column-checkbox {
+    padding: rem(10px) $spacing-05;
+  }
+
+  // -------------------------------------
+  // Th > Button
+  // -------------------------------------
+  .#{$prefix}--table-sort-v2 {
+    @include button-reset(false);
+    position: relative;
+    font: inherit;
+    cursor: pointer;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 100%;
+    color: $text-01;
+    padding: 0 $spacing-04;
+  }
+
+  .#{$prefix}--table-sort-v2:focus {
+    outline: 0;
+  }
+
+  .#{$prefix}--table-sort-v2:focus svg,
+  .#{$prefix}--table-sort-v2:hover svg {
+    opacity: 1;
+  }
+  // -------------------------------------
+  //Th > Button > Svg (Sort Icons)
+  // -------------------------------------
+
+  .#{$prefix}--data-table-v2 th:hover .#{$prefix}--table-sort-v2 .#{$prefix}--table-sort-v2__icon {
+    opacity: 1;
+  }
+
+  // inactive icons
+  .#{$prefix}--table-sort-v2 .#{$prefix}--table-sort-v2__icon-inactive {
+    display: block;
+  }
+
+  .#{$prefix}--table-sort-v2 .#{$prefix}--table-sort-v2__icon {
+    display: none;
+  }
+
+  .#{$prefix}--table-sort-v2__icon-inactive {
+    position: relative;
+    left: rem(2px);
+    transition: transform $transition--base $carbon--standard-easing;
+    transform: rotate(0);
+    opacity: 0;
+    fill: $ui-05;
+    height: $layout-01;
+    width: auto;
+    min-width: $layout-01;
+  }
+
+  // active icons
+  .#{$prefix}--table-sort-v2.#{$prefix}--table-sort-v2--active .#{$prefix}--table-sort-v2__icon-inactive {
+    display: none;
+  }
+
+  .#{$prefix}--table-sort-v2.#{$prefix}--table-sort-v2--active .#{$prefix}--table-sort-v2__icon {
+    display: block;
+    opacity: 1;
+  }
+
+  .#{$prefix}--table-sort-v2--ascending .#{$prefix}--table-sort-v2__icon {
+    transform: rotate(180deg);
+    transition: transform $transition--base $carbon--standard-easing;
+  }
+
+  .#{$prefix}--table-sort-v2__icon {
+    position: relative;
+    left: rem(2px);
+    transition: transform $transition--base $carbon--standard-easing;
+    transform: rotate(0);
+    opacity: 0;
+    fill: $ui-05;
+    width: auto;
+    min-width: $layout-01;
```

Comment:
What's the `min-width` used for here?

Reply:
These were styles that I copied over from v9 icon and didn't touch. Don't have an answer as to why it was used, but I can go thru and test removing them in the dom to see if they're needed.